spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -789,7 +828,7 @@ definitions:
     short_desc: Deprecated
     public: False
     replaced_by:
-      - MSG_SSR_TILE_DEFINITION
+      - MSG_SSR_TILE_DEFINITION_DEP
```

Comment:
Should these deprecated symbols be `_DEP_A`? Seems to be a common pattern in other deprecated messages